Firestore/Example/Tests/Integration/FSTStreamTests.mm

```diff
@@ -285,11 +285,11 @@ - (void)testStreamClosesWhenIdle {
 
   [_workerDispatchQueue runDelayedCallbacksUntil:FSTTimerIDWriteStreamIdle];
 
-  dispatch_sync(_testQueue, ^{
+  [_workerDispatchQueue dispatchSync:^{
     [_workerDispatchQueue enterCheckedOperation:^{
```

Comment:
I think we're double-calling enterCheckedOperation now (which should hopefully be causing the test to fail?).  Here and below.

Reply:
Yep. Fixed.